examples/README.md

```diff
@@ -0,0 +1,22 @@
+# Example model specifications
+
+## Choosing an example
+
+| Example name   | Description                                       | Runtime       |
+| -------------- | ------------------------------------------------- | ------------- |
+| `long_running` | Consumption-savings model with health and leisure | a few minutes |
+
+## Running an example
+
+Say you want to solve the [`long_running`](./long_running.py) example locally. In a
+Python shell, execute:
+
+```python
+from lcm.entry_point import get_lcm_function
+
+from long_running import MODEL_CONFIG, PARAMS
```

Comment:
Why would this work? Don't I need a path unless my shell is located in that very folder?

Reply:
Correct, it this will only run when you are in the examples folder. This code snippet aims to show from which module to take the model config and params, and what lcm functions to call. This is not supposed to be directly "runnable." Currently, I do not think that I want these examples inside src (which would fix the path issue), because I want them to be standalone. Any suggestions? I could add a warning sign noting that you need to be in the examples folder for this to work?

Reply:
Yeah, should just be part of the instructions.